atest/robot/output/xunit.robot

```diff
@@ -8,6 +8,13 @@ Suite Setup         Run Tests    -x xunit.xml -l log.html --skiponfailure täg
 ${TESTDATA}         misc/non_ascii.robot
 ${PASS AND FAIL}    misc/pass_and_fail.robot
 ${INVALID}          %{TEMPDIR}${/}ïnvälïd-xünït.xml
+${MERGE ONE}        %{TEMPDIR}${/}merge1.xml
+${MERGE TWO}        %{TEMPDIR}${/}merge2.xml
+${STIME}            20211215-12:11:10.456
+${XTIMESTAMP}       2021-12-15T12:11:10.456000
+${ETIME}            20211215-12:13:14.789
+${OPTIONS WITH TIMES}
+...                 -x xunit.xml -l log.html --starttime ${STIME} --endtime ${ETIME}
```

Comment:
Most, if not all, of these variables are used in tests only once. Tests would probably be easier to understand if actual values were used directly.